lib/mix/lib/mix/tasks/release.init.ex

```diff
@@ -226,15 +236,21 @@ defmodule Mix.Tasks.Release.Init do
           echo "ERROR: RPC expects an expression as argument" >&2
           exit 1
         fi
-        rpc "$2"
+
+        if [ -z "$3" ]; then
+          rpc_eval "$2"
+        else
+          shift
+          rpc_call $@
```

Comment:
I think you should always quote it?

```suggestion
          rpc_call "$@"
```

Reply:
IIRC, that is going to quote each individual argument and not the whole thing.